runtime/doc/intro.txt

```diff
@@ -133,7 +133,7 @@ There are three ways to report bugs:
    This is a maillist, you need to become a member first and many people will
    see the message.  If you don't want that, e.g. because it is a security
    issue, please contact any of the current Vim maintainers
-   https://github.com/orgs/vim/people (but not Bram or the vim-dev ML).
+   https://github.com/orgs/vim/people (but not the vim-dev ML).
```

Comment:
Bram's account and the vim-dev ML account used to be listed in this URL.
However, they are now private and not visible for non-vim-org members.
So, the comment in the parentheses is not needed anymore?

Reply:
Then perhaps it's better to change it at all as stated in SECURITY.md? It's all about security messages.